gamemode/core/libs/sh_anims.lua

```diff
@@ -422,7 +422,8 @@ if (SERVER) then
 	-- @see LeaveSequence
 	function playerMeta:ForceSequence(sequence, callback, time, bNoFreeze)
 		hook.Run("PlayerEnterSequence", self, sequence, callback, time, bNoFreeze)
-
+		self:SetLocalVelocity(Vector(0, 0, 0))
```

Comment:
This can easily be replaced by [`vector_origin`](https://wiki.facepunch.com/gmod/Global_Variables#constants) to avoid any creation of a vector object every time this function is called.